sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/CreateJacksonParser.scala

```diff
@@ -39,11 +40,28 @@ private[sql] object CreateJacksonParser extends Serializable {
     jsonFactory.createParser(new InputStreamReader(bain, "UTF-8"))
   }
 
-  def text(jsonFactory: JsonFactory, record: Text): JsonParser = {
-    jsonFactory.createParser(record.getBytes, 0, record.getLength)
+  def text(jsonFactory: JsonFactory, record: Text, encoding: Option[String]): JsonParser = {
```

Comment:
https://github.com/apache/spark/pull/20937#discussion_r178476075 seems missed and other hidden comments too.

Reply:
ah, sure. I thought you missed (because they are hidden). will try to take another look soon too.